src/main/kotlin/icfp2019/Strategy.kt

```diff
@@ -1,6 +1,9 @@
 package icfp2019
 
+import org.jgrapht.Graph
+import org.jgrapht.graph.DefaultEdge
+
 interface Strategy {
     // Given a state, return a list of actions, indexed by Robot id.
-    fun getActions(gameState: GameState): Map<RobotId, List<Action>>
+    fun apply(undirectedGraph: Graph<Node, DefaultEdge>, gameState: GameState, initialPosition: Node): Iterable<Action>
```

Comment:
This is a  bigger change,  instead of taking in. the graph just use `GraphAnalyzer.analyze(gameState)`

Reply:
Also use the `core/Strategy`